docs/docs/plugin-authoring-101.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: How Plugins Work
+title: Plugin Authoring 101
```

Comment:
I'm pretty torn over this title. I'm almost wondering if "Plugin file and naming conventions" is closer. Just not sure what people use as a google search term. @marcysutton any ideas?

Reply:
@shannonbux what will go on the page? If "file and naming conventions" isn't too limiting, that would make sense to me...

Reply:
@marcysutton I wasn't clear enough! It would be the same content as How Plugins Work. I just think "How Plugins Work" is actually a misleading title bc that page mostly just talks about file and naming conventions!

Reply:
https://www.gatsbyjs.org/docs/how-plugins-work/

Reply:
ok resolved this through splitting doc up into smaller pieces @marcysutton

Reply:
Ah, I should have read through the whole file. Thanks, Shannon!